### DISCUSSION TITLE
Intervallo data/ora/minuti per evento di morte

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1010] [Supporto servizi cooperativi]
Il modello relativo agli atti di morte, esempio "Morte_002", prevedono la sezione

![image](https://github.com/user-attachments/assets/e4861a58-389c-4308-adcb-0d29be8cd1e0)

che presenta, in particolare:§
- il campo "Data morte"
- i campi "Data presunta inizio", "Data presunta fine"
che, immaginiamo, siano da intendersi mutuamente esclusivi:
a) se il certificato/avviso di morte consegnato all'usc reca l'indicazione della data di decesso "esatta", deve essere valorizzato il campo "Data morte" (vuoti entrambi i campi "Data presunta inizio", "Data presunta fine")
b) se il certificato/avviso di morte consegnato all'usc reca l'indicazione di un intervallo di date all'interno del quale si è avuto il decesso, devono essere valorizzati i campi "Data presunta inizio", "Data presunta fine" (vuoto il campo "Data morte").

E' corretta la supposizione di cui sopra?
E ancora: quale informazione dovrebbe contenere il campo "Note data e ora presunta"?

Ci pare comunque non sia ancora prevista la possibilità di gestire un intervallo generale di date/ore/minuti per l'evento decesso (esempio: _tra le ore 23 e minuti 30 del 24 agosto 2024 e le ore 01 e minuti 30 del 25 agosto 2025_), situazione certamente ricorrente; è alla vostra attenzione l'estensione della modellazione in questo senso?

E ancora: succede talvolta che il certificato di morte rilasciato dal medico necroscopo riporti indicazione non precisa dell'ora/minuti del decesso (esempio: alle ore 22 _circa_); è alla vostra attenzione l'estensione della modellazione in questo senso?

**Comment by acolosimosogei** [comment]
Salve,

di seguito inserisco le domande nella discussion #1010;

	1)E' corretta la supposizione di cui sopra?

E' corretta

	2)quale informazione dovrebbe contenere il campo "Note data e ora presunta"?

Il campo "Note data e ora presunta" è un campo di testo libero e che quindi può essere valorizzato dall'usc liberamente in modo che possano essere gestiti tutte le casistiche in cui le date e l'ora non sono precise.

Grazie

**Comment by acolosimosogei** [comment]
Buonasera, 
procedo a chiudere la discussion, ricordo che in caso di ulteriori dubbi può essere riaperta.
Grazie,
Assistenza ANSC
